dnsext-iterative/DNS/Iterative/Query/LocalZone.hs

```diff
@@ -1,7 +1,7 @@
 module DNS.Iterative.Query.LocalZone where
 
 -- GHC packages
-import Data.Map (Map)
+import Data.Map.Strict (Map)
```

Comment:
`Data.Map.Map` and `Data.Map.Strict.Map` are the same type, is this change necessary?

Reply:
When I do `grep -r Data.Map */DNS | grep -v Data.Map.Strict`, this change ensures that safe lines are not displayed.